src/hdmf/backends/hdf5/h5tools.py

```diff
@@ -1301,7 +1317,7 @@ def _filler():
                 self.__dci_queue.append(dataset=dset, data=data)
             # Write a regular in memory array (e.g., numpy array, list etc.)
             elif hasattr(data, '__len__'):
-                dset = self.__list_fill__(parent, name, data, options)
+                dset = self.__list_fill__(parent, name, data, expandable, options)
```

Comment:
I think `expandable` should be passed to `__setup_empty_dset__` and `__setup_chunked_dset__` as well so that if a user is iteratively writing a `VectorData` or any other dataset, it can also be expanded.

Reply:
Unless these are already expandable. Is that the case?

Reply:
When I read the tutorial, DataChunkIterator already sets the None for maxshape since it has no way of knowing the minimum size of the array.

Reply:
Got it. It should be fine then. Thanks.